src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -73,26 +73,103 @@
  * &lt;module name="MethodLength"/&gt;
  * </pre>
  * <p>
- * To configure the check so that it accepts methods with at most 60 lines:
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *     public MyClass() {  // constructor (line 1)
+ *          /&#42; line 2
+ *              ...
+ *             line 150  &#42;/
+ *     } // violation, the total number of lines is 151 (more than the default 150 lines)
+ *       // empty lines and comments (both single-line and block comments) are counted towards
+ *
+ *     public void firstExample() { // line 1
+ *         //  line 2
+ *         /&#42;  line 3
+ *             line 4 &#42;/
+ *     } // OK, the total number of lines is 6 (less than the default 150 lines)
```

Comment:
Too verbose.

Make comment like ` // line 5, OK , as it is not target `

Apply same to all other cases like this

Reply:
I made changes to the comments.